modules/base.nix

```
@@ -26,7 +27,10 @@
   };
   # … enable passwordless ‘sudo’ for user
   security.sudo.wheelNeedsPassword = false;
-  programs."${userShell}".enable = true;
+  # doing this the hard way, because
+  # programs.bash.enable = true; raises an error
+  programs.zsh.enable = ("zsh" == userShell);
+  programs.fish.enable = ("fish" == userShell);
```

How about the following?

```suggestion
  programs."${userShell}".enable = ("bash" != userShell);
```

---

Hm.... I'm not getting it, wouldn't that set `programs.bash.enable = false` if `userShell = "bash"`?

Yeah:

```
       - The option definition `programs.bash.enable' in `/nix/store/8x5ns4ar25825rxzv4fh13gkkljirxjk-source/modules/base.nix' no longer has any effect; please remove it.
```

Fuck generality.

---

- [ ] Silly me, you're right. Let's keep the original changes from this PR and move forward with this. Apologies for sidetracking with premature optimisationszzz